pkg/acsengine/params_k8s.go

```diff
@@ -299,6 +302,23 @@ func assignKubernetesParameters(properties *api.Properties, parametersMap params
 			}
 
 			addValue(parametersMap, "enableAggregatedAPIs", kubernetesConfig.EnableAggregatedAPIs)
+
+			if properties.HasWindows() {
+				// Kubernetes packages as zip file as created by scripts/build-windows-k8s.sh
+				// will be removed in future release as if gets phased out (https://github.com/Azure/acs-engine/issues/3851)
+				kubeBinariesSASURL := kubernetesConfig.CustomWindowsPackageURL
+				if kubeBinariesSASURL == "" {
+					kubeBinariesSASURL = cloudSpecConfig.KubernetesSpecConfig.KubeBinariesSASURLBase + k8sComponents["windowszip"]
+				}
+				addValue(parametersMap, "kubeBinariesSASURL", kubeBinariesSASURL)
```

Comment:
Let's also move these 4 params definitions out of `kubernetesbase.t` and into `windowsparams.t`

Reply:
Sure!